modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/toast/commands/OpenToast.es.js

```diff
@@ -79,15 +80,19 @@ function openToast({
 
 	unmountComponentAtNode(container);
 
-	const onClose = () => unmountComponentAtNode(container);
+	const unmount = () => unmountComponentAtNode(container);
+
+	const onCloseFn = (event) => {
```

Comment:
Not a superfan of this... but not sure what would be a better approach while also keeping the `onClick` API...